Shared/sdk/net/bitstream.h

```diff
@@ -452,6 +452,8 @@ enum class eBitStreamVersion : unsigned short
     // 2020-11-10 0x71
     PedEnterExit,
 
+    SetPolygonHeight,
```

Comment:
Better clarification.
```suggestion
    SetColPolygonHeight,
```